components/camel-kafka/src/main/java/org/apache/camel/component/kafka/KafkaFetchRecords.java

```diff
@@ -507,7 +517,7 @@ private boolean isReady() {
         } catch (Exception e) {
             // ignore
             LOG.debug("Cannot check hasReadyNodes on KafkaConsumer client (ConsumerNetworkClient) due to: "
-                      + e.getMessage() + ". This exception is ignored.",
+                            + e.getMessage() + ". This exception is ignored.",
```

Comment:
The format check may be complaining because of this line. If you look at it, you can see that it's different than the previous version.